packages/twenty-front/src/modules/object-record/record-index/components/RecordIndexTableContainerEffect.tsx

```diff
@@ -68,5 +74,37 @@ export const RecordIndexTableContainerEffect = () => {
     );
   }, [setRecordCountInCurrentView, setOnEntityCountChange]);
 
+  const setViewFieldsAggregateOperations = useRecoilCallback(
+    ({ set, snapshot }) =>
+      (viewField: ViewField) => {
+        const aggregateOperationForViewField = snapshot
+          .getLoadable(
+            aggregateOperationForViewFieldState({
+              viewFieldId: viewField.id,
+            }),
+          )
+          .getValue();
+
+        if (aggregateOperationForViewField !== viewField.aggregateOperation) {
+          set(
+            aggregateOperationForViewFieldState({
+              viewFieldId: viewField.id,
+            }),
+            viewField.aggregateOperation,
+          );
+        }
+      },
+    [],
+  );
+
+  useEffect(() => {
+    currentViewWithSavedFiltersAndSorts?.viewFields.forEach((viewField) => {
```

Comment:
Ok if it can be undefined then 

if (isDefined(currentViewWithSavedFiltersAndSorts)) {
      for (const viewField of currentViewWithSavedFiltersAndSorts.viewFields) {
        setViewFieldsAggregateOperations(viewField);
      }
    }

It makes the check more explicit